code/sonalyze/sonarlog/postprocess.go

```diff
@@ -127,6 +126,41 @@ func PostprocessLog(
 		*stream = es[:good+1]
 	}
 
+	// For each stream, clean up the gpumem_pct and gpumem_gb fields based on system information, if
+	// available.
+	if configs != nil {
```

Comment:
Does the config file need to be enforced for caching?  And if so, where do we do it?  Does this not mean that the config file becomes an argument to more commands?  And will this not filter into the daemon code?

Reply:
Indeed a config file isn't *required* for anything unless we ask for relative fields in load or jobs.  Code everywhere goes out of its way to work even without the config.  This is annoying...